CONTRIBUTING.md

```diff
@@ -32,9 +30,15 @@ However, most of the things here listed are very generic and apply when contribu
 
 ## Topics
 
-* [Reporting Issues](#reporting-issues)
-* [Submitting Pull Requests](#submitting-pull-requests)
-* [Find bad changes with git bisect](#find-bad-changes-with-git-bisect)
+- [Contributing to the Containers Projects](#contributing-to-the-containers-projects)
+  - [Topics](#topics)
```

Comment:
This links to the top-level title of the file, and to this section itself.

That’s redundant and unnecessary (OTOH if there is a linter complaining and would be shut up by this, I can live with it).

Applies to all three CONTRIBUTING docs.

Reply:
It was my vscode auto formatter that did this, I can certainly look to disable that functionality. I agree that it is a bit weird to like to the same section and top level again.